stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/typing/TypingIndicator.kt

```diff
@@ -0,0 +1,47 @@
+package io.getstream.chat.android.ui.typing
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import io.getstream.chat.android.ui.databinding.StreamTyppingIndicatorViewBinding
+
+public class TypingIndicator : LinearLayout {
+
+    private lateinit var binding: StreamTyppingIndicatorViewBinding
+
+    public var message: String
+        get() = binding.tvUserTyping.text.toString()
+        set(value) {
+            binding.tvUserTyping.text = value
+        }
+
+    public constructor(context: Context?) : super(context) {
+        init()
+    }
+
+    public constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs) {
+        init()
+    }
+
+    public constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(
+        context,
+        attrs,
+        defStyleAttr
+    ) {
+        init()
+    }
+
+    public constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int, defStyleRes: Int) : super(
+        context,
+        attrs,
+        defStyleAttr,
+        defStyleRes
+    ) {
+        init()
+    }
+
+    private fun init() {
+        binding = StreamTyppingIndicatorViewBinding.inflate(LayoutInflater.from(context), this, true)
```

Comment:
Could this be joined with property declaration and declared as `val` instead of `lateinit var`?

Reply:
Let me check...

Reply:
done